activestorage/db/update_migrate/20190731033524_add_service_name_column_to_active_storage_blobs.rb

```diff
@@ -0,0 +1,9 @@
+class AddServiceNameColumnToActiveStorageBlobs < ActiveRecord::Migration[6.0]
+  def up
+    return if column_exists?(:active_storage_blobs, :service_name)
+
+    if table_exists?(:active_storage_blobs)
+      add_column :active_storage_blobs, :service_name, :string, default: "", null: false
```

Comment:
Why is there a default empty string here?

Reply:
You're right. It makes no sense. I've dropped the default value and the null check.